gen/shuffleboard/ShuffleboardContainer.yml

```diff
@@ -14,64 +17,147 @@ classes:
       m_isLayout:
     methods:
       ShuffleboardContainer:
+        # this is abstract, so can't construct it without a trampoline
+        ignore: true
       GetComponents:
       GetLayout:
         overloads:
           std::string_view, BuiltInLayouts:
+            return_value_policy: reference_internal
           std::string_view, LayoutType&:
+            return_value_policy: reference_internal
           std::string_view, std::string_view:
+            return_value_policy: reference_internal
           std::string_view:
+            return_value_policy: reference_internal
       Add:
         overloads:
           std::string_view, wpi::Sendable&:
+            return_value_policy: reference_internal
+            param_override:
+              sendable:
+                name: defaultValue
+            cpp_code: |
+              [](ShuffleboardContainer *self, py::str &key, std::shared_ptr<wpi::Sendable> sendable) -> frc::ComplexWidget& {
+                if (!sendable) {
+                  throw FRC_MakeError(err::NullParameter, "{}", "value");
+                }
+
+                // convert key to a raw string so that we can create a StringRef
```

Comment:
Doesn't pybind11 support string_view arguments?

Reply:
Yes, but I need to store a py::str in the shuffeboard data anyways, so this makes it so I don't have to recreate it.